### PR TITLE
Migrate to rewrite

### PR DESCRIPTION
**Pull request by RDIL**

### CODE REVIEW

**Comment by RDIL** [comment]
@blizma can you please take a second look at this please ❤️

**Comment by blizma** [comment]
ok

**Comment by RDIL** [comment]
Thanks!

File: main.py [hunk 1]
```diff
@@ -50,7 +50,7 @@ def func_tokens():
 
 github.enable_console_debug_logging()
 g = github.Github(j[1])
-client = discord.Client()
+client = discord.AutoShardedClient(shards=20)
```
Comment (blizma) [inline main.py]: IIRC, you don't need to put the amount of shards, and also it's ```AutoShardedBot```

Comment (RDIL) [reply main.py]: Gotta fix that then, thanks

**Comment by RDIL** [comment]
It is in `club.cakebot.ServerUtil`

**Comment by blizma** [comment]
Gotcha.

**Comment by RDIL** [comment]
no worries, I don't like `from` statements either

**Comment by RDIL** [comment]
Going to test on discord once I get home, then merge

**Comment by blizma** [comment]
alright.

**Comment by RDIL** [comment]
no need to worry about flake8

**Comment by RDIL** [comment]
cheese

**Comment by RDIL** [comment]
ok hopefully it should stop dismissing your reviews now

**Comment by RDIL** [comment]
@blizma ```
Traceback (most recent call last):
  File "main.py", line 53, in <module>
    client = discord.AutoShardedBot()
AttributeError: module 'discord' has no attribute 'AutoShardedBot'
```

**Comment by RDIL** [comment]
Aha @blizma I was right: https://discordpy.readthedocs.io/en/latest/api.html?highlight=autosharded#discord.AutoShardedClient

**Pull request merged**